predictions/multilabel_model/multilabel_pred_on_wavfile.py

```diff
@@ -0,0 +1,126 @@
+from predictions.multilabel_model import multilabel_pred
+
+def predict_on_embedding(embedding, config_datas):
+    '''
+    Predict on single embedding for audio clip
+    '''
+    prediction_probs = []
+    prediction_rounded = []
+
+
+    config_data = config_datas
+
+    # Clear keras session before all predictions with model
+    K.clear_session()
+
+    model = multilabel_pred.load_model(config_data["networkCfgJson"],
+                                                    config_data["train"]["outputWeightFile"])
+
+    ##############################################################################
+          # Predict on test data
+    ##############################################################################
+
+    if embedding.shape[0] < 10:
+        # Pad by repeating the last second in embedding till we get to 10 seconds
+        padding = np.ones((10-embedding.shape[0], embedding.shape[1]))*embedding[-1,:]
+        embedding = np.vstack([embedding, padding])
+
+    # If the clip is longer then 10 seconds, then predict on multiple 10-second
+    # windows within the clip. Take the max probability across all windows.
+    pred_prob = 0
+    for index in np.arange(10,embedding.shape[0]+1):
+        windowed_embedding = embedding[(index-10):index,:].reshape((1,10*128,1))
+        pred_prob = max(pred_prob, model.predict(windowed_embedding).ravel()[0])
+
+    # Clear keras session after all predictions with model
+    K.clear_session()
+
+    pred_round = np.round(pred_prob)
+    pred_prob = pred_prob * 100
+
+    prediction_probs.append(pred_prob)
+    prediction_rounded.append(pred_round)
+
+    return prediction_probs, prediction_rounded
+
+def read_config(filepath):
+    with open(filepath,"r") as f:
+        config = json.load(f)
+    return config
+
+def main(predictions_cfg_json, path_for_wavfile):
+
+    ##############################################################################
+            # read the dataframe with feature and labels_name column
+    ##############################################################################
+    EMBEDDINGS = generate_before_predict_BR.main(path_for_wavfile, 0, None, None)
+
+    ##############################################################################
+            # Import json data
+    ##############################################################################
+    #CONFIG_DATAS = get_results_binary_relevance.import_predict_configuration_json(predictions_cfg_json)
+    CONFIG_DATAS = [read_config(file) for file in read_config(predictions_cfg_json)]
+    #LABEL_NAMES = list(CONFIG_DATAS.keys())
+
+    ##############################################################################
+          # Implementing using the keras usual training techinque
+    ##############################################################################
+
+    CLF2_TRAIN_PREDICTION = []
+    CLF2_TRAIN_PREDICTION_PROB = []
+    for data in CONFIG_DATAS:
+        for each_embedding in [EMBEDDINGS]:
```

Comment:
Invert the ordering of the loops
```suggestion
    for each_embedding in [EMBEDDINGS]:
        for data in CONFIG_DATAS:
```